unit-tests/tests.h

```diff
@@ -0,0 +1,104 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#ifndef TESTS_H
+#define TESTS_H
+
+#include "../include/librealsense2/rs.hpp"
+
+#define CATCH_CONFIG_MAIN
```

Comment:
We can't assume this. I suggest you wrap it in some sort of `#ifndef NO_CATCH_CONFIG_MAIN` or such (that's the name I used in `algo-common.h` and other places

Reply:
done